src/auth/auth.action.js

```diff
@@ -21,7 +21,7 @@ export const auth = (code, state, navigation) => {
   return dispatch => {
     dispatch({ type: LOGIN.PENDING });
 
-    fetchAccessToken(code, state)
+    delay(fetchAccessToken(code, state), 2000)
```

Comment:
Hmm are you thinking of having `2s` as a specific constant like `loadDelay`? Personally feel with the `delay` method in `utils`, might be better to have the freedom to use any number we like if we wish to delay other async actions.

What do you think?